rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/report/SW360ReportController.java

```diff
@@ -122,7 +130,7 @@ private void getProjectReports(boolean withLinkedReleases, boolean mailRequest,
                 responseJson.addProperty("response", "Project report download link will get send to the end user.");
                 response.getWriter().write(responseJson.toString());
             } else {
-                downloadExcelReport(withLinkedReleases, response, sw360User, module, projectId);
+                downloadExcelReport(withLinkedReleases, response, sw360User, module, projectId, null);
```

Comment:
Consider using default value for new parameters to avoid modifications at multiple places.